src/pages/404.js

```diff
@@ -9,7 +18,14 @@ import {
   SideNavigation,
   TopBar,
 } from '../components';
-import { graphql } from 'gatsby';
+import { AdvancedPageHit } from '../components/advanced-search';
+import Icon, { iconNames } from '../components/icon';
+import { docsIndex, learnIndex } from '../components/search/indices';
+
+const searchClient = algoliasearch(
+  'NQVJGNW933',
```

Comment:
Feels like we should centralize these values somewhere... what do you think would be the most appropriate strategy?